utils/run_in_ga.sh

```diff
@@ -51,6 +51,20 @@ utils/check.sh /opt/daos/bin/vos_tests
 utils/check.sh /opt/daos/bin/dmg
 echo ::endgroup::
 
+echo ::group::Rebuild, use ALT_PREFIX for ofi
+rm -rf /opt/daos/prereq/release/{ofi,mercury} build/external/release/{ofi,mercury*}
+scons --deps-only PREFIX=/opt/daos/dep TARGET_TYPE=release --build-deps=yes DEPS=ofi --jobs $DEPS_JOBS
```

Comment:
(style)  line over 100 characters
(lint) Double quote to prevent globbing and word splitting. [SC2086]